basxconnect/core/views/tag_views.py

```diff
@@ -0,0 +1,118 @@
+import bread
+import htmlgenerator as hg
+from bread import layout as layout
+from bread.layout.components.button import Button
+from bread.layout.components.modal import Modal, modal_with_trigger
+from bread.utils import aslayout, reverse_model
+from bread.views import AddView
+from django import forms
+from django.http import HttpResponseBadRequest, HttpResponseRedirect
+from django.utils.translation import gettext_lazy as _
+from django.utils.translation import ngettext_lazy
+from htmlgenerator import mark_safe
+
+from basxconnect.core import models
+from basxconnect.core.models import Term, Vocabulary
+
+
+class AddTagView(AddView):
+    model = models.Term
+    fields = ["term", "vocabulary"]
+
+    def form_valid(self, *args, **kwargs):
+        ret = super().form_valid(*args, **kwargs)
+        ret["HX-Redirect"] = f"{self.request.GET['next']}&new-tag={self.object.id}"
+        return ret
+
+
+@aslayout
+def bulk_tag_operation_view(request):
+    operation = request.GET["operation"]
+    initial = request.GET.get("new-tag")
+    if operation not in ["add", "remove"]:
+        return HttpResponseBadRequest("invalid GET parameter 'operation'")
+    persons = request.GET.getlist("persons")
```

Comment:
You "could" do

```python
class PersonList(forms.Form):
    persons = forms.ModelMultipleChoiceField(queryset=models.Person.objects.all())
personlist = PersonList(request.GET)
personlist.is_valid()
personlist.cleaned_data["persons"]
```

But just FYI. The form-parsing and validation is often helpful, but in this case it might over-do it ;)